.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I know it doesn't matter but the 8-space indentation is sending me

Reply:
Finally